code/game/turfs/floor_types.dm

```diff
@@ -8,12 +8,6 @@
 	icon = 'icons/turf/mainship.dmi'
 	icon_state = "default"
 
-/turf/open/floor/mainship/check_alien_construction(mob/living/builder, silent = FALSE, planned_building)
-	if(ispath(planned_building, /turf/closed/wall/)) // The Canterbury moves and will leave a hole in space if there's a resin wall.
```

Comment:
This seems worrying

Reply:
This definitely happened in the old shuttle system, but it has been updated since. Definitely needs testing though.

Reply:
The shuttles are still protected by snowflake code; you can't place resin walls inside.

Reply:
https://i.imgur.com/V1NR9tj.png
Tested just to be sure.